src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -77,6 +102,93 @@ class Config:
         Bool('managed_nfsd', default=True),
     )
 
+    @private
+    def name_to_id_converstion(self, name, name_type='user'):
```

Comment:
woops, a typo in the method name 😄 
```suggestion
    def name_to_id_converstion(self, name, name_type='user'):
```